ui/selectors/selectors.js

```diff
@@ -340,6 +341,13 @@ export function getSelectedInternalAccount(state) {
   return state.metamask.internalAccounts.accounts[accountId];
 }
 
+export function isSelectedInternalAccountEth(state) {
+  const { type } = getSelectedInternalAccount(state);
+  return Boolean(
+    type === EthAccountType.Eoa || type === EthAccountType.Erc4337,
+  );
+}
```

Comment:
Lets add some unit test to this method

Reply:
Added some tests with the new `accounts.ts` selectors file